mvpa2/algorithms/group_clusterthr.py

```diff
@@ -0,0 +1,427 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Cluster thresholding algorithm for a group-level searchlight analysis"""
+
+__docformat__ = 'restructuredtext'
+
+__all__ = ['GroupClusterThreshold',  'get_thresholding_map',
+           'get_cluster_sizes']
+
+if __debug__:
+    from mvpa2.base import debug
+
+import os
+import random
+import bisect
+from collections import Counter
+
+import numpy as np
+
+from scipy.ndimage import measurements
```

Comment:
shouldn't we also get import in the mvpa2.suite  guarded for availability of scipy?